src/string.rs

```diff
@@ -337,6 +371,35 @@ impl String {
     Self::new_from_utf8(scope, value.as_ref(), NewStringType::Normal)
   }
 
+  // Allocates a static ExternalOneByteConst and leaks it.
+  // The allocated ExternalOneByteConst can never be deallocated.
+  #[inline(always)]
+  pub const fn create_external_onebyte_const(
+    buffer: &'static [u8],
+  ) -> ExternalOneByteConst {
+    ExternalOneByteConst {
```

Comment:
We have a const is_ascii in deno_core we can steal for this:

```
  /// Compile-time function to determine if a string is ASCII. Note that UTF-8 chars
  /// longer than one byte have the high-bit set and thus, are not ASCII.
  const fn is_ascii(s: &'static [u8]) -> bool {
    let mut i = 0;
    while i < s.len() {
      if !s[i].is_ascii() {
        return false;
      }
      i += 1;
    }
    true
  }
```

Reply:
Copied this over.